gen/go/proto/services/task_svc/v1/helpers.go

```diff
@@ -43,3 +43,9 @@ func (x *UpdateRoomRequest) UpdatesMap() map[string]interface{} {
 
 	return m
 }
+
+func (r *UpdateBedRequest) UpdatesMap() map[string]interface{} {
+	m := make(map[string]interface{})
+
```

Comment:
add a TODO or implement

Reply:
_`/* Empty for later extension and use */`_